client/tests/components/SearchDocuments.spec.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { mount } from 'enzyme';
+import { SearchDocuments } from '../../app/components/documents/SearchDocuments.jsx';
+
+describe('Search Component', () => {
+  let component;
+  const props = {
+    searchForDocuments: sinon.spy(),
+  };
+  beforeEach(() => {
+    component = mount(<SearchDocuments {...props}/>);
+  });
+  afterEach(() => {
+    props.searchForDocuments.reset();
+  });
+
+  it('should find the neccessary dom elements ', () => {
+    expect(component.find('.search-docs').length).to.equal(1);
+    expect(component.find('input').length).to.equal(1);
+  });
+  it('simulates click events', () => {
```

Comment:
Your test description should be what is asserted not what you did. `calls searchForDocuments once` sounds more like it. You update that everywhere else.